process-migration-service/README.md

```diff
@@ -0,0 +1,58 @@
+# Process Instance Migration Service
+
+This service is intended to be deployed as a standalone application. It can be integrated with KIE Process Server API to execute migrations remotely.
```

Comment:
but that would only apply to migration plans and not migrations itself and the app is called Process Instance Migration which would indicate it will migrate and not just store plans.

Reply:
You have a point. Let's make it a requirement in the README.md. However, from the application point of view I will just raise a Warning if no endpoints were configured.

Reply:
fair enough